core/src/main/java/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor.java

```diff
@@ -136,7 +136,7 @@ public boolean hasFreshToken(@Nonnull User user, @Nullable ApiTokenProperty.Toke
      */
     // used by Jelly view
     @Restricted(NoExternalUse.class)
-    public boolean hasMoreRecentlyUsedToken(@Nonnull User user, @Nullable ApiTokenProperty.TokenInfoAndStats legacyStats) {
+    public boolean hasMoreRecentlyUsedToken(@Nonnull User user, @CheckForNull ApiTokenProperty.TokenInfoAndStats legacyStats) {
```

Comment:
```suggestion
    public boolean hasMoreRecentlyUsedToken(@Nonnull User user, ApiTokenProperty.TokenInfoAndStats legacyStats) {
```

Looks like we need to remove this annotation on `legacyStats` also.